app/api/v2/models/store_model.py

```diff
@@ -24,3 +24,12 @@ def get_all_products(self):
             all_products.append(new_product)
 
         return all_products
+
+    def insert_new_product(self,  product_id, product_name, category_id, stock_amount, price, low_inventory_stock):
+        database = self.db
+        curr = database.cursor()
+        query = "INSERT INTO products (product_id,product_name,category_id,stock_amount,price,low_inventory_stock) VALUES (%s,%s,%s,%s,%s,%s);"
+        curr.execute(query, (product_id, product_name, category_id,
+                             stock_amount, price, low_inventory_stock))
+        database.commit()
+        return {"Message": "Product Save succefully"}, 201
```

Comment:
Response message should be something like "Product added successfully"

Reply:
Okay, Will check on that